frontend/src/components/Toast.tsx

```diff
@@ -0,0 +1,50 @@
+import { Alert } from "antd";
+import React, { FunctionComponent } from "react";
+import styled from "styled-components";
+
+const StyledAlert = styled(Alert)<ToastProps>`
+  position: sticky;
+  top: ${({ topMargin }) => (topMargin ? topMargin : "10px")};
+
+  @media (max-width: 900px) {
+    width: 100%;
+  }
+
+  @media (min-width: 901px) {
+    width: 50%;
+    left: 25%;
+  }
+  margin: auto;
+  z-index: 1000;
+`;
+
+interface ToastProps {
+  message?: string;
+  onClose: () => void;
+  topMargin?: string;
+  alertType?: "success" | "info" | "warning" | "error" | undefined;
```

Comment:
I don't think this needs the `| undefined`, since the `?` after `alertType` makes this allowed to be undefined.